src/Document/LocalTransaction.php

```diff
@@ -36,28 +40,39 @@ class LocalTransaction
 
     private $fee;
 
+    private $time;
+
     private $params;
 
     public function __construct(
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This function has 10 parameters, which is greater than the 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS107)